### ISSUE TITLE
O365 family calendar regularly failing to sync

### ISSUE THREAD

**Issue by r3dpan** [jobisoft/EAS-4-TbSync#244]
## Your environment

TbSync version: 4.7
EAS-4-TbSync version: 4.7
Thunderbird version: 115.2.0 (64-Bit)

Periodic synchronisation time: 10 minutes

## Expected behavior
After adding an O365 (family) account to TbSync i expect to be able to reliably sync it's ressources (calendars, tasks & contacts).

## Actual behavior
When manually syncing the chosen resources the sync job fails at the first resource with the error 'status.Sync.111'.
Most of the time even automated sync jobs seem to fail with the exact same error message too (inconsitent behavior).

## Steps to reproduce
1. Create a microsoft account
2. Add an O365 family license to the account and enable the shared family calendar
3. Add the microsoft account to TbSync (server: outlook.office365.com)
4. Enable sync for the shared family calendar (and optional additional ressources)
5. Click 'Sync now' or wait for the automated sync job to trigger

**Comment by jobisoft** [comment]
https://learn.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-ascmd/95cb9d7c-d33d-4b94-9366-d59911c7a060

ERROR 111 : ServerErrorRetryLater - The server encountered an unknown error, the device SHOULD retry later.

Could be, that O365 does not support to sync Family accounts with EAS. 

Nothing I can do.

**Comment by r3dpan** [comment]
> Could be, that O365 does not support to sync Family accounts with EAS.
> 
> Nothing I can do.

Thanks for the information. I was not aware of this.
I only knew that you couldn't just add a family account via adding an Microsoft Office 365 account in TbSync. Instead you had to use a workaround where you manually configure the server address of the added account to 'outlook.office365.com' (webversion of outlook) which worked pretty well in the past.

_Sidenote: I just found out that the current actual url of the webversion of outlook isn't 'outlook.office365.com' anymore. It seems like this got changed to 'outlook.live.com'. I have just changed this in my TbSync account configuration and now i don't get the 'status.Sync.111' anymore. Don't know how long this will work as this isn't officially supported by microsoft tho._
